index/miner/miner.go

```diff
@@ -2,37 +2,34 @@ package miner
 
 import (
```

Comment:
- As mentioned a bit before, this file was partitioned into: `miner.go`, `meta.go`, and `onchain.go`.
`miner.go` has some central coordination code for the index, and on-chain data gathering lives on `onchain.go` and off-chain data gathering lives on `meta.go`.
- Index information is saved/loaded in datastore
- refactors/fixes